.github/ISSUE_TEMPLATE/release-kickoff-beta.md

```diff
@@ -0,0 +1,61 @@
+WooCommerce {RELEASE_VERSION}-beta.{BETA_PATCH_VERSION}
+
+This issue tracks the progress of a single WooCommerce core plugin release iteration that covers the publishing of a single patch version. Patch version in this context refers to `z` in a version following `x.y.z` where `z` can be any of `0-99`, `0-rc.[0-99]`, or `0-beta.[0-99]`.
+
+## Version being released {RELEASE_VERSION}-beta.{BETA_PATCH_VERSION}
+
+The following details are copied from the official [Building and Publishing guide](https://developer.woocommerce.com/docs/contribution/releases/building-and-publishing/). Please check it to make sure these instructions haven't become out of date.
+
+## Pre-Checks
+
+- [ ] **Verify no open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls?q=is%3Aopen+is%3Apr) or [Issues](https://github.com/woocommerce/woocommerce/issues)** for the milestone matching the release being published.
+  - All pull requests tied to the release milestone must be closed, including [backported pull requests](https://github.com/woocommerce/woocommerce/pulls?q=is%3Apr+label%3A%22type%3A+cherry-pick%22) that may need to be merged into other release branches or trunk.
+- [ ] **Check for unresolved ["cherry pick failed" Pull Requests](https://github.com/woocommerce/woocommerce/pulls?q=is:pr+label:%22cherry+pick+failed%22).**
+  - Ensure any such PRs are either expected or manually resolved via another PR.
+- [ ] **Confirm the Stable tag in** `readme.txt` **matches [trunk on WordPress.org](https://plugins.trac.wordpress.org/browser/woocommerce/trunk/readme.txt#L7).**
+  - The value should match the current stable version, not the version being built.
+- [ ] **Ensure [GitHub services](https://www.githubstatus.com/) are fully operational**
+
+## Build WooCommerce
+
+1. **Run the ["Release: Bump version number" workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-bump-version.yml).**
+   - Run from `trunk`.
+   - Choose the type of version you're releasing (`beta`, `rc`, or `stable`).
+   - Enter as *Release branch* the branch you are about to release from (e.g. `release/10.0`).
+   - Review and merge the PR created.
+
+2. **Run the ["Release: Compile changelog" workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-compile-changelog.yml).**
+   - Run from `trunk` and enter the major version number and the intended release date.
+   - **Review and merge the two PRs created** (one for trunk, one for the release branch).
+   - **Ensure the changelog date is correct.**
+
+3. **Build the release ZIP file.**
+   - Run the ["Release: Build ZIP file" workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-build-zip-file.yml) from the release branch.
```

Comment:
For now, I think it's just making sure we always run the workflows from trunk, particularly step 3, in alignment [with our latest docs](https://github.com/woocommerce/woocommerce/commit/43467476fd555bf1ff6f4ba96fa7c10afc70bb94#diff-a91a3843d5cad6d959951971a5867d9be021926e9b191006d65b20d2fe94424a).

Reply:
^ This applies to all release types, though, so needs to be changed elsewhere.

Reply:
Thanks, updated here: 7719b4ca29be139a2d701bd3415ccca9097103f7